src/braket/aws/aws_quantum_job.py

```diff
@@ -81,6 +81,7 @@ def create(
         aws_session: AwsSession | None = None,
         tags: dict[str, str] | None = None,
         logger: Logger = getLogger(__name__),
+        quiet: bool = False,
```

Comment:
Can we update the job decorator [docstring](https://github.com/amazon-braket/amazon-braket-sdk-python/blob/main/src/braket/jobs/hybrid_job.py#L72) to add this param to the list ignored in local mode?